src/govuk/components/task-list/task-list.yaml

```diff
@@ -0,0 +1,207 @@
+params:
+- name: items
+  type: array
+  required: true
+  description: Array of item objects.
+  params:
+- name: classes
```

Comment:
A very small nitpick that these params of `items` should be indented one more so that it's communicated in the yml and therefore in the nunjucks docs via guidance when we get around to writing it up that these relate to the `items` params, not the params for the root component.

Reply:
Opps! Fixed in b7d5ccbe82e36d1353b9a2ed9e571301a3254130, thanks!